user_guide_src/source/concepts/factories.rst

```diff
@@ -0,0 +1,117 @@
+#########
+Factories
+#########
+
+.. contents::
+    :local:
+    :depth: 2
+
+Introduction
+============
+
+Like ``Services``, ``Factories`` are an extension of autoloading that helps keep your code
+concise yet optimal, without having to pass around object instances between classes. At its
+simplest, ``Factories`` provides a common way to create a class instance and access it from
+anywhere. This is a great way to reuse object states and reduce memory load from keeping
+multiple instances loaded across your app.
+
+Anything can be loaded from a Factory, but the best examples are those classes that are used
+to work on or transmit common data. The framework itself makes heavy use of the ``Config``
+Factory to make sure the correct configuration is in use at any time. 
+
+Take a look at ``Models`` as an example. You can access the Factory specific to ``Models``
+by using the magic static method of the Factories class, ``Factories::models()``. Because
+of the common path structure for namespaces and folders, ``Factories`` knows to look for
+files and classes within ``Models``, so you can request a model by its shorthand base name::
```

Comment:
```suggestion
Take a look at ``Models`` as an example. You can access the Factory specific to ``Models``
by using the magic static method of the Factories class, ``Factories::models()``. Because
of the common path structure for namespaces and folders, ``Factories`` know where to look for
files and classes within ``Models``, so you can request a model by its shorthand base name::
```